dist/src/main/java/io/camunda/zeebe/shared/management/ClusterEndpoint.java

```diff
@@ -219,6 +249,85 @@ private static PartitionStateCode mapPartitionState(final State state) {
     };
   }
 
+  private static GetTopologyResponse mapClusterTopology(final ClusterTopology topology) {
+    final var response = new GetTopologyResponse();
+    final List<BrokerState> brokers = mapBrokerStates(topology.members());
+    final TopologyChange change;
+
+    if (topology.pendingChanges().isPresent()) {
+      change = mapOngoingChange(topology.pendingChanges().get());
+    } else if (topology.lastChange().isPresent()) {
+      change = mapCompletedChange(topology.lastChange().get());
+    } else {
+      change = new TopologyChange();
+    }
```

Comment:
:question: The internal `ClusterTopology` has both lastChange and the pendingChange. But the response we defined has only one change. It is either the lastChange or the pendingChange if there is one. Should we change the response to include both similar to the internal one?

Reply:
Yes, let's include both :+1: